src/middleware/index.js

```diff
@@ -69,31 +70,31 @@ const analyticsMiddleware = store => next => action => {
             timestamp: new Date().toString(),
             account_id: wallet.accountId
         })
+        let id = mixpanel.get_distinct_id();
+        mixpanel.identify(id);
```

Comment:
```
mixpanel.get_distinct_id

Returns the current distinct id of the user. This is either the id automatically generated by the library or the id that has been passed by a call to identify().
```

@icerove not sure I understand the idea here. Docs say that `.get_distinct_id` returns ID set by `.identify`

Can you explain big picture of what are you trying to achieve?

Reply:
if I want to set up user profile, I have to call function `identify` and then use function `people.set`. see this: https://developer.mixpanel.com/docs/javascript#setting-profile-properties